api/envoy/config/core/v3/protocol.proto

```diff
@@ -542,8 +542,9 @@ message Http3ProtocolOptions {
   // <https://datatracker.ietf.org/doc/html/rfc8441>`_
   // and settings `proposed for HTTP/3
   // <https://datatracker.ietf.org/doc/draft-ietf-httpbis-h3-websockets/>`_
-  // Note that HTTP/3 CONNECT is not yet an RFC.
-  bool allow_extended_connect = 5 [(xds.annotations.v3.field_status).work_in_progress = true];
```

Comment:
Is the draft now an RFC? Let's either update the link, or reinstate the comment and WIP

Reply:
It's not a RFC yet. Will reinstate the comment. But do we still consider this as WIP?

Reply:
yeah IMO it stays WIP until there's an RFC just in case something changes with wire format or required headers

Reply:
reverted